### DISCUSSION TITLE
LibrOffice table cells containing lists are not exported to markup formats Asciidoc nor reStructuredText

### DISCUSSION THREAD

**Discussion by andrewjmaguire** [jgm/pandoc#9523] [Q&A] [answered]
I have a libreOffice.odt doc that contains tables with cells having text with unnumbered list items in them.
Asciidoc and reStructuredText are able to support this but pandoc is not extracting the text in the unnumbered list at all.
They are also not included when extracting to LaTeX, so it suggests the parsing of LibreOffice is at issue.

Any ideas?

[pandoc-table-test-default.odt](https://github.com/jgm/pandoc/files/14427454/pandoc-table-test-default.odt)

`pandoc --extract-media Media -t asciidoc -s pandoc-table-test-default.odt -o pandoc-table-test-default.adoc.txt`
[pandoc-table-test-default.adoc.txt](https://github.com/jgm/pandoc/files/14427466/pandoc-table-test-default.adoc.txt)

`pandoc --extract-media Media -t rst -s pandoc-table-test-default.odt -o pandoc-table-test-default.rst.txt`
[pandoc-table-test-default.rst.txt](https://github.com/jgm/pandoc/files/14427467/pandoc-table-test-default.rst.txt)

(.txt extension used to allow Github upload)

**Comment by jgm** [comment]
What version of pandoc are you using?  If it's an old one, try with the latest.

**Reply by andrewjmaguire** [reply]
I am using 2.9.2.1 the default version installed from Ubuntu.

In the meantime, I found a workaround by using https://writer2latex.sourceforge.net/index.html to first convert ODT to HTML5 (not LaTeX), then use pandoc to convert HTML to Asciidoc/reStructuredText etc.

I'll try and have a look at the new version, sometime

Thanks for your fast response

**Reply by jgm** [reply]
I think it will work with the new version. Your version is very old.
We offer deb packages that can be installed on Ubuntu.

**Reply by andrewjmaguire** [reply]
Do you supply checksums for the prepackaged binary files? Good to ensure they are authentic :)

**Reply by andrewjmaguire** [reply, marked as answer]
I have tried the latest version 3.1.12.2 and so far can confirm it supports what I needed.

The workaround using Writer2Latex described above, although *it* worked, the use of the older pandoc version failed to process the HTML5 files correctly in a different way.
